doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -1522,6 +1522,104 @@ Numeral notations
     As noted above, the :n:`(abstract after @num)` directive has no
     effect when :n:`@ident__2` lands in an :g:`option` type.
 
+String notations
+-----------------
+
+.. cmd:: String Notation @ident__1 @ident__2 @ident__3 : @scope.
+   :name: String Notation
```

Comment:
The rendering shows a problem: https://coq.gitlab.io/-/coq/-/jobs/122607710/artifacts/_install_ci/share/doc/coq/sphinx/html/user-extensions/syntax-extensions.html#string-notations
BTW, this problem also exists in numeral notations (please fix in a separate PR so that it can be backported): https://coq.gitlab.io/-/coq/-/jobs/122607710/artifacts/_install_ci/share/doc/coq/sphinx/html/user-extensions/syntax-extensions.html#numeral-notations
The solution is just to not provide the `:name:` line.

Reply:
Thanks for pointing this out.  I'm confused, though.  Why does it render fine for `Arguments` (which has
```rst
.. cmd:: Arguments @qualid {+ @name%@scope}
   :name: Arguments (scopes)

   It is possible to set in advance that some arguments of a given constant have
```
) and for `Print Scope` (which has
```rst

   .. cmdv:: Print Scope @scope
      :name: Print Scope

      This displays all the notations defined in the interpretation scope :token:`scope`.
      It also displays the delimiting key if any and the class to which the
      scope is bound, if any.
```
) but not for `Numeral Notation`?

Reply:
Huh. Let me investigate. Not sure why it does this. I would proceed with this PR and open an unrelated bug.

Reply:
Ah, I think I know why. The body of this object isn't indented with 3 spaces, just 2.
If you indent the body with three spaces it might fix it. IIRC, the body of an object and its options need to lign up. If they don't, reST thinks that the :name: is just a continuation line.

Reply:
Thanks!  Now updated

Reply:
Thanks!  Now updated.